stylesheets/components/_button.scss

```diff
@@ -1,3 +1,60 @@
 // -----------------------------------------------------------------------------
 // This file contains all styles related to the button component.
 // -----------------------------------------------------------------------------
+  .button {
+    display: flex;
+    justify-content: center;
+    align-items: center;  
+    flex: 0 0 auto;
+    border-radius: 4px;
+    
+    &:hover {
+        cursor: pointer;
+        box-shadow:(0 0 2px 1px $color-text-footer);
+        color: $color-button-text;
+        background-color: $color-text-footer;        
+      }
+      
+      &--yellow {
```

Comment:
what if color changes?
consider using expanatory names for example button-primary, button-submit which explains the purpose or type of the button

Reply:
completed